### ISSUE TITLE
align-model attribute can cause model to be misaligned with shadow

### ISSUE THREAD

**Issue by cdata** [GoogleWebComponents/model-viewer#520]
When setting `align-model` to `origin`, it's possible (and in fact likely) that the model will become misaligned relative to its shadow.

**Comment by elalish** [comment]
There are some other issues regarding align-model and camera framing. What is the goal of this attribute? My guess for desired behavior when align-model is set to (0,0,0), is that the camera points at the model origin (which the model is not centered at) and frames the scene such that the whole model is still visible from that perspective. I'm glossing over some subtlety regarding floor height, but does this sound generally correct?

**Comment by cdata** [comment]
For background on the attribute, check out #417 and #441 .

TL;DR once the model has been framed based on its absolute extents, it may not be framed subjectively. For example, if a model is asymmetrical on its x-axis, it may appear to be rotating off center if the majority of its visible volume rests on one side or the other of the viewport.

`align-model` is a half-measure that allows a designer to strategically translate a model in a design tool so that it is more subjectively centered on any access after framing.

**Comment by elalish** [comment]
Thanks for the background. Does my description of its desired behavior sound right? Currently, if you set align-model to something other than default, the camera does not reframe, so the object will likely be out of bounds sometimes. That seems wrong to me, and is closely related to why its shadow doesn't follow it.
